implementations/generate.js

```diff
@@ -6,108 +6,75 @@
 /**
  * Generates the HTTP Signatures Implementation Report given
  * a set of *-report.json files.
+ * Expects the report files be the product of your ImplementationReporter.js
  */
 const fs = require('fs');
 const path = require('path');
 const testConfig = require('./test.json');
+const template = require('./handlebars');
 
 // extract the results from all of the test files
-const allTests = [];
-const allResults = {};
 const dirContents = fs.readdirSync(__dirname);
 const files = dirContents.filter(
-  contents => {return contents.match(/.*-report.json/ig);});
+  contents => contents.match(/.*-report.json/ig));
 
 if(!files.length) {
   throw new Error(
     'Failed to find any reports in the form of your_program-report.json ' +
     `in the dir ${__dirname}`);
 }
 
-function getTestStatus(test, pendingTitles) {
-  if(pendingTitles.includes(test.fullTitle)) {
-    return 'skipped';
+//masterName is the name of the main report
+//used to get the title of all tests for the spec
+function makeMaster(masterName) {
+  const context = {
+    implementations: [],
+    tests: {}
+  };
+  const masterFile = `${masterName}-report.json`;
+  const masterPath = files.find(f => f === masterFile);
+  if(!masterPath) {
+    throw new Error(`Unable to find json report at ${masterPath}
```

Comment:
The error message using backticks like this is going to have extra whitespace.  Please change to use + on line continuation.

Reply:
I don't mind the extra white space especially in a giant scroll of test results, but addressed this in https://github.com/w3c-dvcg/http-signatures-test-suite/pull/5/commits/f791a4746422086bc8a91bda4d1292eb935eea51